### PR TITLE
Updating namespace names as per the review

### PR DESCRIPTION
**Pull request by bsomeshekar**
Updating namespace names as per the review
https://github.com/Azure/azure-sdk-pr/issues/1889

# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

**Review by msyyc — APPROVED** [review]
Approve for Python and @bsomeshekar you also need approval from other language owners.

**Review by tadelesh — APPROVED** [review]

**Comment by tadelesh** [comment]
Approved for Go.

**Review by qiaozha — APPROVED** [review]

File: specification/edge/Microsoft.Edge.Management/tspconfig.yaml [hunk 1]
```diff
@@ -7,6 +20,47 @@ options:
...
+    experimentalExtensibleEnums: true
+    enableOperationGroup: true
+    flavor: azure
+    package-dir: "arm-commonedgesitemanageroperations"
+    packageDetails:
+      name: "@azure/arm-commonedgesitemanageroperations"
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.ResourceManager.CommonEdgeSiteManagerOperations"
+    namespace: "{package-dir}"
+    emitter-output-dir: "{csharp-sdk-folder}/sdk/{service-directory-name}/{namespace}/src"
```
[the remark this answers is not in the document]
Comment (bsomeshekar) [reply specification/edge/Microsoft.Edge.Management/tspconfig.yaml]: Removed

File: specification/edge/Microsoft.Edge.Sites.Management/tspconfig.yaml [hunk 2]
```diff
@@ -7,6 +20,48 @@ options:
...
+    experimentalExtensibleEnums: true
+    enableOperationGroup: true
+    flavor: azure
+    package-dir: "arm-sitemanager"
+    packageDetails:
+      name: "@azure/arm-sitemanager"
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.ResourceManager.SiteManager"
+    namespace: "{package-dir}"
+    emitter-output-dir: "{csharp-sdk-folder}/sdk/{service-directory-name}/{namespace}/src"
```
[the remark this answers is not in the document]
Comment (bsomeshekar) [reply specification/edge/Microsoft.Edge.Sites.Management/tspconfig.yaml]: Removed emitter-output-dir

**Review by live1206 — APPROVED** [review]

**Pull request merged**